docs/configuration.md

```diff
@@ -35,8 +35,20 @@ val conf = new SparkConf()
 val sc = new SparkContext(conf)
 {% endhighlight %}
 
-Note that we can have more than 1 thread in local mode, and in cases like spark streaming, we may actually
-require one to prevent any sort of starvation issues.
+Note that we can have more than 1 thread in local mode, and in cases like Spark Streaming, we may 
+actually require one to prevent any sort of starvation issues.
+
+Properties that specify some time duration should be configured with a unit of time. 
+The following format is accepted:
+ 
+    5s (seconds)
+    25ms (milliseconds)
+    10m or 10min (minutes)
+    3h (hours)
+    5d (days)
+    1y (years)
+
+If no units are provided, the default unit specified in the documentation is assumed. 
```

Comment:
I wonder whether we should actually say this. The whole point of doing this is to deprecate the old way of specifying these configs such that we don't have to maintain the docs to include the old default units.